dataops/requirements-onprem.txt

```diff
@@ -0,0 +1 @@
+data-pipelines-cli[docker,datahub,git,postgres,dbt-all]==0.25.1
```

Comment:
I'm not sure I understand the purpose of specifying `postgres` again when it has already been specified in `dbt-all` for reference. Could you please clarify? The link for reference is https://github.com/getindata/data-pipelines-cli/blob/develop/setup.py#L30.